.github/workflows/flowzone.yml

```diff
@@ -19,3 +19,4 @@ jobs:
       DOCKERHUB_TOKEN: ${{ secrets.DOCKER_REGISTRY_PASS }}
     with:
       docker_images: balena/open-balena-vpn
+      runs_on: '["ubuntu-20.04"]'
```

Comment:
Is this necessary?

Reply:
Yeah, any recent systemd-based OS cannot run systemd in a container. This is going to be true of any of our components using open-balena-base that run the container as part of testing.
See: https://github.com/moby/moby/issues/42275
See: https://balena.zulipchat.com/#narrow/stream/345882-_help/topic/flowzone.20builder.20circle.20CI.20still.20going/near/314681108